### PR TITLE
Crear componente Header para Backoffice

### PR DESCRIPTION
**Pull request by MartinCaparros**
Side bar created for backoffice navigation, it opens onClick on menu icon and close on leftArrow icon onClick.

User Story: https://alkemy-labs.atlassian.net/browse/OT91-123

Evidence:

Captura de pantalla 2021-11-23 120904

Captura de pantalla 2021-11-23 120920

### CODE REVIEW

File: src/Components/Backoffice/BackofficeHeader.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    }),
+    ...(open && {
+      width: `calc(100% - ${drawerWidth}px)`,
+      marginLeft: `${drawerWidth}px`,
+      transition: theme.transitions.create(['margin', 'width'], {
+        easing: theme.transitions.easing.easeOut,
+        duration: theme.transitions.duration.enteringScreen,
+      }),
+    }),
+  }));
```
Comment (saralavanchy) [inline src/Components/Backoffice/BackofficeHeader.js]: move this components to a file for each of them on Utils components folder

File: src/Components/Backoffice/BackofficeHeader.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+
```
Comment (saralavanchy) [inline src/Components/Backoffice/BackofficeHeader.js]: this extra space is not necessary, please delete it

File: src/Components/Backoffice/BackofficeHeader.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        </Typography>
+      </AppBar>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+          width: drawerWidth,
+          boxSizing: 'border-box',
+        },
```
Comment (saralavanchy) [inline src/Components/Backoffice/BackofficeHeader.js]: move this styles into a style file

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Backoffice/BackofficeHeader.js (hunk 1 above)
Comment (MartinCaparros) [reply src/Components/Backoffice/BackofficeHeader.js]: Solved!

File: src/Components/Backoffice/BackofficeHeader.js (hunk 2 above)
Comment (MartinCaparros) [reply src/Components/Backoffice/BackofficeHeader.js]: Done

File: src/Components/Backoffice/BackofficeHeader.js (hunk 3 above)
Comment (MartinCaparros) [reply src/Components/Backoffice/BackofficeHeader.js]: I think its the better way for styling MUI components, at least its how it appear on the documentation. For better css code we need to create a theme that affects all the application.

Comment (saralavanchy) [reply src/Components/Backoffice/BackofficeHeader.js]: you can continue having this like it is and moving it to other file.

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**